models/api.models.js

```diff
@@ -0,0 +1,9 @@
+const db = require('../db/connection')
+const fs = require('fs/promises')
+
+exports.findApis = () => {
+  return fs.readFile('./endpoints.json', 'utf-8').then((apiData)=> {
+    const parsedApiData =  JSON.parse(apiData)
+    return parsedApiData
```

Comment:
Nice! This can also be done by requiring the file contents into this file, similarly to how you have done in your tests